### DISCUSSION TITLE
proxies not working

### DISCUSSION THREAD

**Discussion by nondualit** [qxxa/instahunt#9] [General]
Hi there,

I tried different proxies in these formats, but it keeps telling me the proxies don't work—even though they do! I'm currently using one in my browser right now.

42 usernames in usernames.txt file
Use proxy? [y/n]: y
Checking proxies.txt for usable proxies...
Testing HTTPS proxy: https://156.253.176.143:3128
SSL verification failed for proxy https://156.253.176.143:3128: Cannot connect to host 156.253.176.143:3128 ssl:default [[SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1000)]
Valid proxies: 0

some  proxies.txt  in the file
156.253.176.143:3128
104.207.59.185:3128

anyone?

**Comment by nondualit** [comment]
this are not free procies are paid at proxt scraper

**Comment by qxxa** [comment]
Hi @nondualit ,

Make sure you're using residential proxies. Also, instead of HTTPS, try using the SOCKS5 protocol.
